proposal/S413/README.rst

```diff
@@ -0,0 +1,122 @@
+Basic Information
+-------------------
+Github Repositroy: TBD
+
+https://github.com/S413/AllThatJazz
+
+Problem to Solve
+-----------------
+Harmony Search algorithm is a heuristic optimization algorithm.
+It is inspired from the method jazz musicians employ when playing
+a piece, in order to find the best harmoics. The algorithm stores
+a series of 'harmonies', and at each iteration generates a new one.
+This new harmony is compared to the worst harmony in memory, and if
+it is judged superior, the harmony memory is updated.
+
+The algorithm has seen much use since its initial publication in 2001,
+so much so that many modified versions have been created;
+on the other hand, it has also been under harsh criticism. 
+
+In any case, there seem to be few options for those wanting to use the
+algorithms in python. 
+
+Prospective Users
+------------------
+Anyone with an interest in optimization problems. 
```

Comment:
There are many who are interested in numerical optimization and build a career upon it.